source/scss/_pf-component-card.scss

```diff
@@ -24,3 +24,29 @@
     border-top-color: $pf-card-border-color
   }
 }
+.pf-card--accented {
+  border-top: 2px solid $color-pf-blue-300;
+}
+.pf-card--multiselect__block,
+.pf-card--multiselect__header {
+  padding-left: $pf-spacer-xxl;
+  padding-right: $pf-spacer-xxl;
+}
+.pf-card__checkbox {
+  position: absolute;
+  top: $pf-spacer-xxxs;
+  left: $pf-spacer-xxs;
+  input[type=checkbox] {
+    //visibility: hidden; is not keyboard accessible and should be replaced with something that keeps the checkbox "visible" in the dom but not rendered on the screen
```

Comment:
Let's document this. Can we create a mixin so we can reuse the same solution to hide/show elements in different components?